it/v6/lossClaim/amendType/def1/Def1_AmendLossClaimTypeISpec.scala

```diff
@@ -144,19 +147,26 @@ class Def1_AmendLossClaimTypeISpec extends IntegrationBaseSpec {
                               requestClaimId: String,
                               requestBody: JsValue,
                               expectedStatus: Int,
-                              expectedBody: MtdError): Unit = {
+                              expectedBody: MtdError,
+                              requestTaxYearClaimedFor: String = "2020-21",
+                              requestDownstreamTaxYearClaimedFor: String = "20-21"): Unit = {
```

Comment:
The downstream isn't influenced by the input data so we don't need this as a parameter here.